monai/handlers/post_processing.py

```diff
@@ -49,8 +49,5 @@ def __call__(self, engine: Engine) -> None:
         Args:
             engine: Ignite Engine, it can be a trainer, validator or evaluator.
         """
-        engine.state.batch, engine.state.output = engine_apply_transform(
-            batch=engine.state.batch,
-            output=engine.state.output,
-            transform=self.transform,
-        )
+        for i, (b, o) in enumerate(zip(engine.state.batch, engine.state.output)):
```

Comment:
what's the type of `engine.state.batch` and `output`? probably need a check here